### PR TITLE
resource/alicloud_ess_scalinggroup_vserver_groups: remove clientToken for same error code

### PR DESCRIPTION
**Pull request by fuliu-zln**

### CODE REVIEW

**Review by ChenHanZhang — COMMENTED** [review]
approved

File: alicloud/resource_alicloud_ess_scalinggroup_vserver_groups.go [hunk 1]
```diff
@@ -263,7 +263,6 @@ func attachVserverGroups(d *schema.ResourceData, client *connectivity.AliyunClie
 		if err != nil {
 			return WrapError(err)
 		}
-		request["ClientToken"] = buildClientToken(action)
```
Comment (ChenHanZhang) [inline alicloud/resource_alicloud_ess_scalinggroup_vserver_groups.go:266]: 为啥把幂等参数去掉了

Comment (fuliu-zln) [reply alicloud/resource_alicloud_ess_scalinggroup_vserver_groups.go:266]: 见AONE:https://aone.alibaba-inc.com/v2/project/528766/req#viewIdentifier=fca5df829f51a2ae696f7c4b&openWorkitemIdentifier=53743134
评论

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Pull request merged**